docs/sdk/push-notifications/fragments/ios/messaging-campaign.md

```diff
@@ -72,37 +72,37 @@ The following steps show how to receive push notifications targeted for your app
 
     **Note**: If you already have push notification delegate methods, you can just add the `interceptDidRegisterForRemoteNotifications` and `interceptDidReceiveRemoteNotification` callbacks to Pinpoint client.
 
-1. In Xcode Project Navigator, choose your app name at the top, choose your app name under **Targets**, choose the **Capabilities** tab, and then turn on **Push Notifications**.
+2. In Xcode Project Navigator, choose your app name at the top, choose your app name under **Targets**, choose the **Capabilities** tab, and then turn on **Push Notifications**.
 
-1. Configure the app to run in the **Release** profile instead of the default **Debug** profile. Perform the following steps to get a notification to the device:
```

Comment:
heh, I actually prefer using 1. versus monotonically increasing because Markdown parses will increment the number automatically. The benefit is for a future write refactoring or adding steps. Suggestion: Leave them all 1.

Reply:
Completely agree. I'm intrigued because I didn't change them manually, so what happened here? 🤔🤔

Changed it back to 1.

Reply:
Sounds like your editor was being helpful!

Reply:
I reviewed all the ordered lists in this folder. We had a mix of things. Mainly the formatting was not done properly in order to rely on `1.` (e.g. code snippets belonging to items in the list). I fixed them and converted every ordered list to all '1.'s